### ISSUE TITLE
Expand access to recruitment dashboard

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#604]
# TL;DR
Allow researchers to access the participant analytics / recruitment dashboard view.

# Narrative
As a researcher, I would like to be able to evaluate the success of my recruitment strategies over time.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Researchers with access to participant data can see that data in the analytics view
- [ ] Participant analytics permission is preserved for Lookit staff who should be able to see all data in analytics

# Implementation Notes
Note that the analytics view is currently rather resource-intensive; it may need to be streamlined to allow for many researchers using it instead of just occasional use by staff. Carefully review DB calls and server-side processing.
